stdlib/format.mli

```diff
@@ -1610,3 +1686,5 @@ val kasprintf : (string -> 'a) -> ('b, formatter, unit, 'a) format4 -> 'b
   using {{: ppx_deriving.show} https://github.com/ocaml-ppx/ppx_deriving}
   or similar ppx derivers.
 *)
+
+
```

Comment:
It is better to reduce noises by removing those accidental additional blank lines.